### DISCUSSION TITLE
Keyball61 右と左の特定行が反応しない

### DISCUSSION THREAD

**Discussion by akki-F** [Yowkees/keyball#369] [Q&A] [answered]
keyball61組み立て後、写真の赤枠行のキーが反応しません。
付近のダイオードの再はんだとりつけや、ソケットもはんだしなおしてみましたが解決しませんでした。
テンプラの可能性があるダイオードを教えていただけないでしょうか？
その他原因になる箇所ありましたらご指摘いただけますと幸いです。
![keyball右](https://github.com/Yowkees/keyball/assets/45144201/f90c2ede-b035-44cf-b819-6bb2e2ec580b)
![keyball左](https://github.com/Yowkees/keyball/assets/45144201/a49d4188-ec66-48ae-8da4-ae8d50305685)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

横一列すべて動かない場合、一番可能性が高いのはProMicro裏の半田付けになります。
ROWで始まる表記がある端子が怪しいです。

写真を見ると、半田量が少なめですので、少し足してみてください。
※ただ、足しすぎ注意です！スルーホールに半田が吸い込まれているように感じたらすぐに中止してください。半田量は充分で、裏に流れ込んでいる可能性があります。

その他、ProMicro裏の半田付けで隣とショートしている端子がないかもご確認いただければ幸いです。

以上、お手数ですがご対応お願いいたします。

**Reply by akki-F** [reply]
ご回答ありがとうございます。
指摘通りProMicro裏の半田付け対応したところ問題なく動作するようになりました。
大変助かりましたありがとうございます。

**Comment by Yowkees** [comment]
無事に修理できたようで良かったです。安心しました。
ぜひ末永く楽しんでいただければ幸いです。
